### PR TITLE
[CCR] Do not unnecessarily wrap fetch exception in a ElasticSearch exception and

### PR DESCRIPTION
**Pull request by martijnvg**
properly map fetch_exception.exception field as object.

The extra `caused_by` level is not necessary here:

```
"fetch_exceptions": [
              {
                "from_seq_no": 1,
                "retries": 106,
                "exception": {
                  "type": "exception",
                  "reason": "[index1] IndexNotFoundException[no such index]",
                  "caused_by": {
                    "type": "index_not_found_exception",
                    "reason": "no such index",
                    "index_uuid": "_na_",
                    "index": "index1"
                  }
                }
              }
            ],
```

### CODE REVIEW

**Comment by martijnvg** [comment]
Just [changed](8ee1afdbafe4a8fd715a58d3f5b44269f35ffe98) exception field mapping to be a type nested by request from @ycombinator

**Review by jasontedor — APPROVED** [review]
LGTM.

File: x-pack/plugin/core/src/main/resources/monitoring-es.json [hunk 1]
```diff
@@ -991,7 +991,15 @@
                   "type": "integer"
                 },
                 "exception": {
-                  "type": "text"
+                  "type": "nested",
```
Comment (ycombinator) [inline x-pack/plugin/core/src/main/resources/monitoring-es.json]: Hey @martijnvg, I was actually wondering if `fetch_exceptions` (not `exception`) should be `nested` as it contains an array of objects. Thoughts?

Comment (martijnvg) [reply x-pack/plugin/core/src/main/resources/monitoring-es.json]: @ycombinator I pushed: 1797b73aba2b2e5f7edea503b47ae2b4df8299da

**Review by ycombinator — APPROVED** [review]
The mapping change for `fetch_exceptions` (not the main intent of this PR) LGTM.

**Pull request merged**